packages/automerge-repo-network-messagechannel/src/index.ts

```diff
@@ -1,24 +1,14 @@
-import {
-  ChannelId,
-  NetworkAdapter,
-  NetworkAdapterEvents,
-  PeerId,
-} from "automerge-repo"
-import debug from "debug"
-import EventEmitter from "eventemitter3"
+import { ChannelId, NetworkAdapter, PeerId } from "automerge-repo"
 import { MessagePortRef } from "./MessagePortRef"
 import { StrongMessagePortRef } from "./StrongMessagePortRef"
 import { WeakMessagePortRef } from "./WeakMessagePortRef"
 
-const log = debug("messagechannel")
+import debug from "debug"
+const log = debug("ar:messagechannel")
```

Comment:
this should be a full string: "automerge-repo"

Reply:
✅